.github/workflows/build-complete-samples.yml

```diff
@@ -416,6 +416,10 @@ jobs:
               name: 'bot-adaptive-card-actions'
```

Comment:
I can't see deployed manifest for this sample.

Also, link that manifest with main readme file.

Reply:
Fixed